### ISSUE TITLE
500 Internal Server Error when clicking Create without a Customer ID 

### ISSUE THREAD

**Issue by pkutlesa** [DevOps-Agile-Squad/orders#101]
**As a** user 
**I need** the application to catch errors when clicking the Create button without a Customer ID 
**So that** the server doesn't crash and I can correctly create orders 

**Assumptions:** 
* Add error handling to ```POST /orders``` 

**Acceptance Criteria:**
```
Given a running application
When I press the Create button with an empty Customer ID field 
Then the server doesn't crash with a 500 internal server error and I receive a useful error message such as "a Customer ID is required to create an Order" 
```

**Comment by pkutlesa** [comment]
This issue will be tackled after refactoring to Flask-RESTX. The refactored Create route should throw a 400 Bad Request Error if the posted data is not valid.

**Comment by pkutlesa** [comment]
This bug was resolved with the completion of issue #90
